AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomMaterialEditor_BasicTests.py

```diff
@@ -27,12 +25,10 @@
 TEST_MATERIAL_2 = "002_BaseColorLerp.material"
 TEST_MATERIAL_3 = "003_MetalMatte.material"
 TEST_DATA_PATH = os.path.join(
-    azlmbr.paths.engroot, "Gems", "Atom", "TestData", "TestData", "Materials", "StandardPbrTestCases"
-)
+    azlmbr.paths.engroot, "Gems", "Atom", "TestData", "TestData", "Materials", "StandardPbrTestCases").lower()
```

Comment:
This looks like it will break on Linux, because there is no way this path is all lower case on Linux. What was the root cause of the error that prompted this change?